src/mixins/common-mixin.ts

```diff
@@ -86,37 +86,22 @@ export default class CommonMixin extends Vue {
 
   /** The appropriate edit label for corrections, alterations, change or conversion filings. */
   get editLabel (): string {
-    switch (true) {
-      case this.isCorrectionFiling: return 'Correct'
-
-      case this.isAlterationFiling:
-      case this.isChangeRegFiling:
-      case this.isConversionFiling: return 'Change'
-    }
+    if (this.isCorrectionFiling) return 'Correct'
+    if (this.isAlterationFiling || this.isChangeRegFiling || this.isFirmConversionFiling) return 'Change'
     return 'Edit' // should never happen
   }
```

Comment:
I saw this switch statement (and the ones below) and thought maybe we should use if statements instead. This now looks so much better. 👍

Reply:
Yeah.

The other style may be sexy but this style is easier to read and understand, so I changed my mind 😁

Reply:
😄 👍